frontend/vue/components/Syllabus/SyllabusLayout.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+  <div class="syllabus">
+    <section class="syllabus__title-container">
+      <!-- TODO: provide translations for all strings -->
+      <h1 class="syllabus__title">
+        Classroom
+      </h1>
+      <!-- TODO: update section subtitle -->
+      <p class="syllabus__title-text">
```

Comment:
Same comment as `SyllabusCreateLayout`

To me, this class is confusing because the title is text too. Maybe `syllabus__title__subtitle` or `syllabus__title__description` could be more descriptive. What do you think?